packages/api/portfolioDrafts/submit/persistPortfolioDraft.ts

```diff
@@ -0,0 +1,65 @@
+import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { Context } from "aws-lambda";
+import { dynamodbDocumentClient as client } from "../../utils/aws-sdk/dynamodb";
+import { NO_SUCH_PORTFOLIO_DRAFT_404 } from "../../utils/errors";
+import { DynamoDBException, DatabaseResult } from "../../utils/response";
+import { ProvisioningStatus } from "../../models/ProvisioningStatus";
+import { ValidatedPortfolioDraft } from "./validateCompletePortfolioDraft";
+
+const ATAT_TABLE_NAME = process.env.ATAT_TABLE_NAME ?? "";
+
+/**
+ * Updates the status of the Provisioning Portfolio Draft at the end of
+ * being processed by the Step Functions
+ *
+ * @param stateInput - the input from the previous Portfolio Draft Validation
+ *  Task in the Step Function
+ */
+export async function handler(successfulStateInput: ValidatedPortfolioDraft, context?: Context): Promise<void> {
+  const successfulPortfolioDraft = successfulStateInput;
+  const portfolioDraftId = successfulPortfolioDraft.id;
+  console.log("SFN INPUT (successful): " + JSON.stringify(successfulStateInput));
+  const databaseResult = await updatePortfolioDraftStatus(
+    ATAT_TABLE_NAME,
+    portfolioDraftId,
+    ProvisioningStatus.COMPLETE
+  );
+  console.log("DB UPDATE RESULT (successful): " + JSON.stringify(databaseResult));
+}
+
+export async function updatePortfolioDraftStatus(
+  tableName: string,
+  portfolioDraftId: string,
+  status: string
+): Promise<DatabaseResult> {
+  const now = new Date().toISOString();
+  try {
+    return await client.send(
+      new UpdateCommand({
+        TableName: tableName,
```

Comment:
```
return await client.send(
      new UpdateCommand({
        TableName: process.env.ATAT_TABLE_NAME ?? "",
```
To be consistent I would rewrite this to include the TableName inline. See `createPortfolioStep.ts` for an example of how this done.